.mockery.yaml

```diff
@@ -0,0 +1,11 @@
+with-expecter: false
+inpackage: True
+dir: "./pkg/tangy"
+mockname: "Mock{{.InterfaceName}}"
+outpkg: "{{.PackageName}}"
+filename: "{{.InterfaceName}}_mock.go"
+all: True
+packages:
+  github.com/content-services/tang:
+    config:
```

Comment:
under config add, `filename: "tangy_mock.go"` so the file name is `tangy_mock` instead of `Tangy_mock`

Reply:
Will do!